G/GAP/common.jl

```diff
@@ -0,0 +1,29 @@
+include("../../L/libjulia/common.jl")
+
+function gap_platforms(; expand_julia_versions::Bool=false)
+    if expand_julia_versions
+        platforms = vcat(libjulia_platforms.(julia_versions)...)
+
+        # we only care about 64bit builds
+        filter!(p -> nbits(p) == 64, platforms)
+
+        # Windows is not supported
+        filter!(!Sys.iswindows, platforms)
+
+        return platforms
+    else
+        # TODO: create the platforms from the above list coming from libjulia
+        # by removing the julia_version tag
+        
+        platforms = supported_platforms()
+        filter!(p -> nbits(p) == 64, platforms) # we only care about 64bit builds
+        filter!(!Sys.iswindows, platforms)      # Windows is not supported
```

Comment:
How about moving this common code out to the end of the function, i.e. after the `if/else/end` have
```
    filter!(p -> nbits(p) == 64, platforms) # we only care about 64bit builds
    filter!(!Sys.iswindows, platforms)      # Windows is not supported
    return platforms
```

Reply:
Good point, I'll do that when I redo parts of this function. cf. https://github.com/JuliaPackaging/Yggdrasil/pull/11412#discussion_r2149401738